packages/es-components/src/components/containers/tabPanels/TabPanel.js

```diff
@@ -58,7 +58,7 @@ class TabPanel extends React.Component {
   render() {
     const { children } = this.props;
     const elements = React.Children.map(children, (child, i) => {
-      const isSelected = child.props.name === this.state.value;
+      const isSelected = child.props.name.key === this.state.value.key;
```

Comment:
Maybe I'm missing something, but I don't think child.props.name or this.state.value have .key properties on them.